src/theme/components/product/ProductListingCard.vue

```diff
@@ -55,10 +59,16 @@ const props = defineProps<{
 
 const { formatPrice } = useCurrency()
 const { navigateToI18n } = useLocalisation()
+const wishlistStore = useWishlist()
```

Comment:
Implementation code is the same as in product detail buybox, so we should move it in a separate ProductAddToWishlist component and just drop it on listing and detail